tests/ios/unit-test/test-imports-app/BUILD.bazel

```diff
@@ -20,7 +20,7 @@ ios_application(
         "main.m",
     ],
     bundle_id = "com.example.TestImports-App",
-    families = ["ipad"],
+    families = ["ipad", "iphone"],
```

Comment:
why this change? it's good to test that we correctly handle the non-default set of families

Reply:
✅  Removed in the latest rev - seemed related to [default test runner selection changes](https://github.com/bazel-ios/rules_ios/pull/188#discussion_r557606756)